app/templates/views/performance.html

```diff
@@ -15,7 +15,7 @@
     </div>
   </div>
 
-  <h2 class="govuk-heading-m">
+  <h2 class="govuk-heading-m" id='messages-since-may-2016'>
```

Comment:
```suggestion
  <h2 class="govuk-heading-m" id='messages-sent-since-may-2016'>
```
This should match the heading text so it's guessable, like the others.

Reply:
Thanks Tom, have updated!